packages/patternfly-4/react-core/src/components/AboutModal/AboutModalContainer.js

```diff
@@ -66,8 +68,10 @@ const ModalContent = ({
   ...props
 }) => {
   if (!isOpen) {
+    document.body.classList.remove(css(styles.backdropOpen));
```

Comment:
`document` and `window` do not exist in SSR (server side rendering) scenarios. this will need to be moved to `componentDidMount` and `componentDidUpdate`

Reply:
as another side note, it might be helpful to try this in page layout. I do not believe that the body is what scrolls there. I think it is the main section. Setting overflow here would actually not do anything in an app that uses our page layout component.

Reply:
@dmiller9911 Correct, in a page layout the section scrolls not the body, so this would not be an issue there